akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/StashBuffer.scala

```diff
@@ -81,6 +80,23 @@ import akka.japi.function.Procedure
    */
   def forEach(f: Procedure[T]): Unit
 
+
+  /**
+   * Tests whether this [[StashBuffer]] contains a given message.
+   *
+   * @param message the message to test
+   * @return true if the buffer contains the element, false otherwise.
+   */
+  def contains[U >: T](message:U): Boolean
+
+  /**
+   * Tests whether a predicate holds for at least one element of this [[StashBuffer]].
+   *
+   * @param predicate the predicate used to test
+   * @return true if the predicate holds for at least one element, false otherwise.
+   */
+  def anyMatch(predicate:Predicate[T]): Boolean
```

Comment:
I choose `anyMatch` because that's what we have in the Java Stream API. That's their equivalent to `exists`. 

Which is convenient for us, as it doesn't conflict with the scala `def exist(predicate: T => Boolean)`

Reply:
Any change to use it?

Reply:
I don't get your question. Let me explain the concrete use case I had to motivate it.

I was building something for which  I regular get an message from the outside whenever an update occurs. For instance, a message like `StuffGotUpdated` is sent when something needs to be processed.

The actor needs to react to such update message and it spawns an async computation. While waiting for it, it needs to stash incoming message. At the end, the stash may be empty or it may contain one or many `StuffGotUpdated`. If it has many, the actor should only trigger one async computation. One is enough, not many. 

So, with an API to check if a message is already in the stash, I can ensure that only one `StuffGotUpdated` is ever stashed and only one async computation is triggered.